.circleci/run.sh

```diff
@@ -102,6 +102,10 @@ coverage()
     make -j$N -C ../druntime -f posix.mak MODEL=$MODEL
     make -j$N -C ../phobos -f posix.mak MODEL=$MODEL
 
+    # FIXME
+    # Temporarily the failing long file name test has been removed
+    rm -rf test_results/compilable/issue17167
```

Comment:
I think here you meant:
`test_results/compilable/issue17167.sh`

Reply:
Yes. Sorry :/